src/Synapse/OAuth2/OAuthController.php

```diff
@@ -95,7 +101,8 @@ public function __construct(
         AccessTokenMapper $accessTokenMapper,
         RefreshTokenMapper $refreshTokenMapper,
         Mustache_Engine $mustache,
-        Session $session
+        Session $session,
+        array $loginConfiguration
```

Comment:
If we're only extracting `requireVerification` from this, might as well just pass that in to keep things tidy.